tests/Countries/UzbekistanTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Spatie\Holidays\Tests\Countries;
+
+use Carbon\CarbonImmutable;
+use Spatie\Holidays\Holidays;
+
+it('can calculate uzbekistan holidays', function ($year) {
+    CarbonImmutable::setTestNow($year.'-01-01');
+
+    $holidays = Holidays::for(country: 'uz')->get();
+
+    expect($holidays)
+        ->toBeArray()
+        ->not()->toBeEmpty();
+
+    expect(formatDates($holidays))->toMatchSnapshot();
+})->with([2000, 2006, 2023, 2024, 2033]);
```

Comment:
Only add years you have verified to be correct

Reply:
Removed unnecessary tests and verified all snapshots as well